### PR TITLE
Get the production code and the develop code to be the same code base

### PR DESCRIPTION
**Pull request by dunkgray**

### CODE REVIEW

File: dags/nci_ard/nci_s2_ard.py [hunk 1]
```diff
@@ -35,7 +36,16 @@
...
+# Probably should have a more stable variable to use.
+if (
+    environ.get("AIRFLOW__WEBSERVER__BASE_URL")
+    == "https://airflow.sandbox.dea.ga.gov.au"
+):
+    # Production
+    schedule_interval = "0 19 * * *"
+else:
+    # develop
+    schedule_interval = None
```
Comment (jmettes) [inline dags/nci_ard/nci_s2_ard.py:48]: Does this need dry run too?

**Review by jmettes — APPROVED** [review]

**Comment by jmettes** [comment]
For posterity: this logic relies on the `AIRFLOW__WEBSERVER__BASE_URL` environment variable being set. Currently, due to the way we deploy Airflow, this is currently true. But in theory this could not be true in the future.

In that case, the code:

```
if (
    environ.get("AIRFLOW__WEBSERVER__BASE_URL")
    == "https://airflow.sandbox.dea.ga.gov.au"
):
    do_prod_stuff
else:
    do_dev_stuff
```

the environment variable will be null, and luckily won't result in prod stuff accidentally being run in dev. However, in prod it will run dev code (with dry-run) and will silently fail despite Airflow tasks still passing, making it harder to notice the issue.

**Comment by dunkgray** [comment]
>However, in prod it will run dev code (with dry-run) and will silently fail despite Airflow tasks still passing, making it harder to notice the issue.
My understanding is schedule will become None so both processes will be turned off, without any 'red columns' or new green columns.

File: dags/nci_ard/nci_s2_ard.py (hunk 1 above)
Comment (dunkgray) [reply dags/nci_ard/nci_s2_ard.py:48]: I might do that next time I'm testing in dev.

**Pull request merged**